### ISSUE TITLE
Errore dal log di render

### ISSUE THREAD

**Issue by SbxGus** [UrloMythus/MammaMia#10]
Ciao, ho visto che ieri hai fatto un update ed ho provato a fare un fork per istanza privata. 
Premetto che ne avevo già fatta una precedentemente, e inizialmente funzionava correttamente, poi i risultati sono via via andati scemando fino ad arrivare praticamente a 0.
Quindi oggi ho rifatto istanza con la nuova build, ma qualunque serie vado a cercare, render mi torna errore.

Ti lascio il log nel caso ti possa essere utile.

Handling movie or series: tt11691774:3:8
1
2021
<Response [403]>
MammaMia: StreamingCommunity failed Expecting value: line 1 column 1 (char 0)
INFO:     37.117.122.153:0 - "GET /%7CSC%7C/stream/series/tt11691774%3A3%3A8.json HTTP/1.1" 404 Not Found

**Comment by UrloMythus** [comment]
Yep, SC ha aggiunto Cloudfare. Per ora non è funzionante a meno di usare l'addon in locale

**Comment by SbxGus** [comment]
Sì, in local funziona perfettamente. Però, hostando da pc-win non riesco a farla raggiungere dagli altri dispositivi, per esempio la tv.
Esiste un modo che puoi indicarmi a grandi linee?

**Comment by UrloMythus** [comment]
> Sì, in local funziona perfettamente. Però, hostando da pc-win non riesco a farla raggiungere dagli altri dispositivi, per esempio la tv. Esiste un modo che puoi indicarmi a grandi linee?
> 
> Ho provato a fare un node server con local-ip.co con tutta la buona volontà e sono riuscito a far andare il loro indirizzo prova, poi però provando cambiando host e port nel tuo config non va e non ho idea a questo punto se è il metodo giusto, o se sto sbagliando qualcosa io nell'implementarlo.
> 
> Comunque, ti ringrazio a prescindere.

L'addon per essere raggiungibile da altri dispositivi deve essere in HTTPS, in teoria il cambio porta dovrebbe funzionare così come l'host, però non sono sicuro che sia quello il modo giusto per farlo funzionare con local IP. Potresti provare con un server nginx
